torchao/prototype/scaled_grouped_mm/kernels/test_jagged_float8_scales.py

```diff
@@ -11,7 +11,7 @@
     triton_fp8_col_major_jagged_colwise_scales,
     triton_fp8_row_major_jagged_rowwise_scales,
 )
-from torchao.prototype.scaled_grouped_mm.scaled_grouped_mm import (
+from torchao.prototype.scaled_grouped_mm.test.utils import (
```

Comment:
nit: this feels a little weird that we are importing test utils for kernels

Reply:
Can you clarify? This is in the test_*.py file, so I am importing the test utils for unit tests.

Reply:
IMO if I want to run tests I run `pytest test/...` that is why I was confused, as though it isn't as bad to ship extra dirs/libs in python still not the best practice.  Feel pretty strongly that tests should be in `test` folder but everything else don't really care lol

Reply:
Sounds good, I moved the tests to `ao/test/prototype/scaled_grouped_mm/...`